src/Context/ContentContext.php

```diff
@@ -290,6 +290,12 @@ protected function createContent($contentType, array $data): void
                 case '_hidden':
                     $locationStruct->hidden = (bool)$value;
                     break;
+                case '_sortField':
+                    $locationStruct->sortField = (int)$value;
```

Comment:
Shouldn't this be a string?

Reply:
Internally it is all handled as int. But I changed the implementation so that the features use strings according to the constants defined in the Location class.